source/configuration_manual/config_file/config_variables.rst

```diff
@@ -54,6 +56,38 @@ If :ref:`var_expand_crypt_plugin` is loaded, these also work globally:
 |                               | .. dovecotadded:: 2.2.29    |
 +-------------------------------+-----------------------------+
 
+.. _variables-defaults:
+
+Defaults
+^^^^^^^^
+
+Dovecot supports a list of default values, that can be accessed using the
+syntax ``%{dovecot:<key>}``. The following keys are supported:
+
++---------------+-------------------------------------------------------------+
+| Key           | Value                                                       |
++===============+=============================================================+
+| name          | Name of distributed package                                 |
+|               | (Default: ``Dovecot``)                                      |
++---------------+-------------------------------------------------------------+
+| version       | Dovecot version                                             |
++---------------+-------------------------------------------------------------+
+| os            | OS name reported by uname syscall                           |
+|               | (similar to ``uname -s`` output)                            |
++---------------+-------------------------------------------------------------+
+| os-version    | OS version reported by uname syscall                        |
+|               | (similar to ``uname -r`` output)                            |
```

Comment:
os & os-version seem like they belong to the system: variable that Aki is adding in the other merge request.

Also I'm not sure uname is a syscall. It could be just \`\`uname()\`\`?

Reply:
Rebased, moved and reworded!